keyboards/keychron/q2/ansi_encoder/rules.mk

```diff
@@ -20,6 +20,7 @@ ENCODER_ENABLE = yes        # Enable Encoder
 DIP_SWITCH_ENABLE = yes
 RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = CKLED2001
+LTO_ENABLE = yes
```

Comment:
Is there a reason why we need LTO? Firmware size doesn't seem to be an issue...

Reply:
All right, i consider deleting them. thanks